src/scene/materials/standard-material.js

```diff
@@ -865,7 +872,8 @@ class StandardMaterial extends Material {
 
         // Minimal options for Depth and Shadow passes
         const shaderPassInfo = ShaderPass.get(device).getByIndex(pass);
-        const minimalOptions = pass === SHADER_DEPTH || pass === SHADER_PICK || shaderPassInfo.isShadow;
+        const minimalOptions = this._useDisplacement ? false : pass === SHADER_DEPTH || pass === SHADER_PICK || shaderPassInfo.isShadow;
```

Comment:
this seems to be the only use of `_useDisplacement` - should the `_` be there? If it works without it, do we need this change at all? (maybe I'm just missing something)

Reply:
Was thinking about changing it to private at first, then drifted. `_` gonna be removed.